app/views/equipment_models/show.html.erb

```diff
@@ -147,6 +147,32 @@
        <p><%= Requirement.list_requirement_admins(current_user, @equipment_model) %></p>
   </section>
 <% end %>
+<% if can? :manage, EquipmentModel %>
+  <section id="pending_reservations">
+      <h2>Pending Reservations</h2><hr />
+
+      <div class="row">
+        <div class="col-md-3">
+          <h4>Today</h4>
+        </div>
+        <div class="col-md-3">
+          <h4>Next 7 Day</h4>
+        </div>
+      </div>
+      <div class="row giant-numbers">
+        <div class="col-md-3">
+          <%= @today.count %> 
+        </div>
+        <div class="col-md-3">
+          <%= @upcoming.count %>
+        </div>
+      </div>
+
+      <p id="pending_reservations">
```

Comment:
We should only include the table if `@today.count + @upcoming.count > 0`, but we should include the section regardless just to specify that nothing's coming up in the next week.